vshard/storage/init.lua

```diff
@@ -251,7 +251,17 @@ end
 -- exceptions are not allowed.
```

Comment:
> NO_TEST=<already tested>

But where?

Reply:
It was in commit msg, but github suppresses everything inside <...> in PR's comment. However, this is not important now, as I added the test, as it turns out this problem also happens with functions, created with `box.schema.func.create(..., {body = " ... "})`